python/cudf/cudf/tests/test_repr.py

```diff
@@ -41,9 +41,7 @@ def test_null_series(nrows, dtype):
 
     pd.options.display.max_rows = int(nrows)
     psrepr = repr(ps)
-    psrepr = psrepr.replace("NaN", "<NA>")
-    psrepr = psrepr.replace("NaT", "<NA>")
-    psrepr = psrepr.replace("None", "<NA>")
+    psrepr = psrepr.replace("NaN", "<NA>").replace("None", "<NA>")
```

Comment:
Lines 44 and 72 are splitting the computation differently. One calls `repr` first, then replaces. The other calls `repr` and replaces inline. I'd prefer sticking to one of those options.

Reply:
Changed to make it consistent with the rest of the file.